pipenv/cli/options.py

```diff
@@ -288,6 +289,11 @@ def callback(ctx, param, value):
 
 def three_option(f):
     def callback(ctx, param, value):
+        secho(
```

Comment:
You should move this down inside the `if value is not None:` and then your tests will pass.  Right now it will always alert users, even if not passing this flag.

Reply:
Done.